src/transformState.js

```diff
@@ -4,8 +4,28 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
   // write code here
+  for (const action in actions) {
+    switch (actions[action].type) {
```

Comment:
`actions` is array. so which method we should use to run thorough the array?

```suggestion
  for (const action of actions) {
    switch (action.type) {
```

Reply:
use this approach everywhere